rickshaw/main.py

```diff
@@ -20,25 +20,24 @@ def main(args=None):
     p.add_argument('-rh', dest='rh', action="store_true", help='runs the simulations after they have been generated')
     p.add_argument('-v', dest='v', action="store_true", help='verbose mode will pretty print generated files')
     ns = p.parse_args(args=args)
-    simspec = {}
+    temp_spec = {}
```

Comment:
non-blocking: `temp_spec` is not a great variable name because it isn't super descriptive.